website/docs/verifiable-document/comparison.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Comparison
+sidebar_label: Comparison
+---
+
+In this section, we will discuss the differences between Ethereum Smart Contract Verifiable Credentials and DID Verifiable Credentials. The goal is to provide enough information for you to choose the correct workflow depending on your needs.
+
+## Infrastructure
+
+By using Ethereum Smart Contracts, you will of course be bound to the Ethereum infrastructure. Every transaction will be stored in the Blockchain. Forever. You won't need to maintain any infrastructure on your own. On top of that you will be able to monitor every transaction that happened, and detect abnormal activities. For instance if one user manage to steal your wallet private key, you will find evidence directly un the Blockchain.
+
+Regarding DID, you won't need to maintain any infrastructure as well... or maybe you will need. It will depend on your needs. Indeed, No records of your transactions will be made by default. DID will work straight, out of the box. However, if you want to keep track on every transaction, you will have to do it yourself. One could ask: "Why would I need to track every transaction ?". Unlike for Ethereum Smart Contracts, if a user manage to steal your signing private key, you will never be aware of it. Keeping track of every signed document could be a way to counter this issue.
```

Comment:
```suggestion
Regarding DID, you won't need to maintain any infrastructure as well... or maybe you will need. It will depend on your needs. Indeed, No records of your transactions will be made by default. DID will work straight, out of the box. However, if you want to keep track on every transaction, you will have to do it yourself. One could ask: "Why would I need to track every transaction ?". Unlike for Ethereum Smart Contracts, if someone manages to steal your signing private key, you will never be aware of it. Keeping track of every signed document could be a way to counter this issue.
```

Reply:
keeping track of every signed document does not help you to counter this issue though, the malicious party can still issue and you will never know unless you come across a fake one AND compare it to your records

Reply:
To delete the sentence